### PR TITLE
OFL-1.1 optionally templated

### LINKED ISSUE #451: OFL-1.1 standard license header
We [currently don't declare `standardLicenseHeader` for the OFL-1.1][1], but they define one in their license document (above the title):

```
$ curl -sL 'http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web' |
>   w3m -T text/html -dump | grep -A7 'Copyright (c)' | cut -b 15-
Copyright (c) <dates>, <Copyright Holder> (<URL|email>),
with Reserved Font Name <Reserved Font Name>.

Copyright (c) <dates>, <additional Copyright Holder> (<URL
|email>),
with Reserved Font Name <additional Reserved Font Name>.

Copyright (c) <dates>, <additional Copyright Holder> (<URL|
email>).

This Font Software is licensed under the SIL Open Font License,
Version 1.1.

This license is copied below, and is also available with a FAQ
at: http://scripts.sil.org/OFL
```

And they *also* [discuss their recommendations for using it in their FAQ][2]:

> * Put your copyright and Reserved Font Names information at the beginning of the main OFL.txt file in place of the dedicated placeholders (marked with the `<>` characters). Include this file in your release package.
> * Put your copyright and the OFL text with your chosen Reserved Font Name(s) into your font files (the copyright and license fields). A link to the OFL text on the OFL web site is an acceptable (but not recommended) alternative. Also add this information to any other components (build scripts, glyph databases, documentation, test files, etc). Accurate metadata in your font files is beneficial to you as an increasing number of applications are exposing this information to the user. For example, clickable links can bring users back to your website and let them know about other work you have done or services you provide. Depending on the format of your fonts and sources, you can use template human-readable headers or machine-readable metadata. You should also double-check that there is no conflicting metadata in the font itself contradicting the license, such as the fstype bits in the os2 table or fields in the name table.

The first point suggests they intend the full copyright/license to be used as you'd use the GPL-3.0 license text (for example), by dropping it in a project level file.

The “Also add this information to any other components (build scripts…” in the second point suggests they intend the full copyright/license to also be their standard license header.  Our current XML schema doesn't provide an easy way to declare that the `standardLicenseHeader` is the same as the license text, but maybe it should?  Perhaps via some generic XML reference mechanism?  Otherwise we're going to have to duplicate this information in both places.

[1]: https://github.com/spdx/license-list-XML/blob/9984ebd50cf621010d9b24b95fa7dd47e42ba79a/src/OFL-1.1.xml
[2]: http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL#83afb016

### PR DESCRIPTION
**Pull request by mlinksva**
See previous discussion at #451

See https://opensource.org/licenses/OFL-1.1 and instructions at http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL#83afb016 and many in the wild at https://github.com/search?q=license%3Aofl-1.1

### CODE REVIEW

File: src/OFL-1.1.xml [hunk 1]
```diff
@@ -6,6 +6,11 @@
          <crossRef>http://www.opensource.org/licenses/OFL-1.1</crossRef>
       </crossRefs>
       <notes>This license was released: 26 February 2007.</notes>
+      <optional>
```
Comment (wking) [inline src/OFL-1.1.xml]: `copyrightText` is already optional ([guideline 10.1.1][1]).  I'm in favor of explicit `<optional>` tags for optional copyright text, but that's not where this repo is now.  I think we should shift this line after `<copyrightText>` for this PR and move it back to here it if/when the whole repository moves to use explicit `<optional>` tags for `<copyrightText>`.

[1]: https://spdx.org/spdx-license-list/matching-guidelines

File: src/OFL-1.1.xml [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+         <copyrightText><p>Copyright (c) &lt;dates&gt;, &lt;Copyright Holder&gt; (&lt;URL|email&gt;),
+            <br/>with Reserved Font Name &lt;Reserved Font Name&gt;.</p></copyrightText>
+         <p>This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is copied below, and is also available with a FAQ at: http://scripts.sil.org/OFL</p>
```
Comment (wking) [inline src/OFL-1.1.xml]: The “This license is copied below…” bit is a separate paragraph [upstream][1].

[1]: http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web

File: src/OFL-1.1.xml (hunk 1 above)
Comment (mlinksva) [reply src/OFL-1.1.xml]: Thanks, agreed on all points.

**Comment by mlinksva** [comment]
OK, I removed 'fixes'.

**Pull request merged**